packages/frontend/src/hooks/useFeeTokens.ts

```diff
@@ -0,0 +1,45 @@
+import type { SupportedChain } from "@metrom-xyz/contracts";
+import { useChainId } from "wagmi";
+import { metromApiClient } from "../commons";
+import { useEffect, useState } from "react";
+import type { FeeToken } from "@metrom-xyz/sdk";
+
+export function useFeeTokens(): {
+    loading: boolean;
+    tokens: FeeToken[];
+} {
+    const chainId: SupportedChain = useChainId();
+
+    const [tokens, setTokens] = useState<FeeToken[]>([]);
+    const [loading, setLoading] = useState(false);
+
+    useEffect(() => {
```

Comment:
I think we can use react query for the hook.

Reply:
Addressed in 0b5a3975dc77e2b23d8f88310a3ac92c4fb2b11f